packages/core/src/repository/data-source/sql/raw-sql.data-source.ts

```diff
@@ -280,7 +284,7 @@ export class RawSQLDataSource implements GetDataSource<RawSQLData>, PutDataSourc
         const insertionIds = await this.sqlInterface.transaction((sqlInterface: SQLInterface) => {
             return Promise.all(
                 values.map((value, idx) => {
-                    let id = value[BaseColumnId];
+                    let id = value[BaseColumnId] as number;
```

Comment:
Is it always a number? Couldn't it be a string?

Reply:
This was discussed in Slack and we decided to keep it simple for now: https://mobilejazz.slack.com/archives/CG3BP4799/p1637311784009000